inception-project/inception
# #2046 List logged events with spring websockets

**What's in the PR**
- added spring websocket endpoint, channel for logged events
- added vue component to subscribe to event channel and show events

**How to test manually**
* Add `websocket.enabled=true` and `websocket.loggedevents.enabled=true` to your settings.properties
* Be a admin user
* Click on the feed-icon in the footer bar to see recent events
* Be notified by info feedback messages when event occurs

**Automatic testing**

**Documentation**
